capture/reader-libpcap-file.c

```diff
@@ -179,6 +182,9 @@ LOCAL void reader_libpcapfile_init_monitor()
 LOCAL int reader_libpcapfile_process(char *filename)
 {
     char         errbuf[1024];
+    char 				 path [256];
+    char         *p;
+    struct 			 stat stats;
```

Comment:
style nic pick, please change to "struct stat    stats", where the stats lines up with the p